modules/Components/src/client/ACASFormFields.coffee

```diff
@@ -785,6 +789,82 @@ class ACASFormLSFileValueFieldController extends ACASFormAbstractFieldController
 		@$('.bv_deleteSavedFile').hide()
 		@createNewFileChooser()
 
+
+class ACASFormLSBlobValueFieldController extends ACASFormLSFileValueFieldController
+	###
+		Launching controller must:
+		- Initialize the model with an LSValue
+    - Provide allowedFileTypes
+    Do whatever else is required or optional in ACASFormAbstractFieldController
+	###
+
+	initialize: (options)->
+		super(options)
+		@fileURL = "/blobUploads"
+
+	arrayToBase64String: (a)->
+		btoa new Uint8Array(a).reduce(((data, byte) ->
+			data + String.fromCharCode(byte)
+		), '')
+
+	setupFileController: ->
+		fileValue = @getModel().get('value')
+		if @isEmpty()
+			@createNewFileChooser()
+			@$('.bv_deleteSavedFile').hide()
+		else
+			displayText = @getModel().get('comments')
+			if !displayText?
+				displayText = "InternalErrorUnknownFileName"
+			id = @getModel().get('id')
+			# Display inline not supported right now because guessing mimetype from extension isn't available
+			# so always be falsey here but leaving this in place
+			if false and @displayInline and @mimeType?
+				@$('.bv_file').html '<img src="data:' + @mimeType + ';base64,'+@arrayToBase64String(fileValue)+'" alt="'+ displayText+'">'
+				@$('.bv_file').html '<img src="/api/thingvalues/downloadThingBlobValueByID/'+id+'" alt="'+ displayText+'">'
```

Comment:
Why are we setting the html to two different values on two consecutive lines? This seems like we're missing another if/else?

Reply:
addressed in a new commit by removing the second line (this whole block is not used as noted above but its good to remove so later it won't cause confusion)